kafka-streams-framework/src/main/java/org/hypertrace/core/kafkastreams/framework/async/RecordToForward.java

```diff
@@ -17,4 +17,9 @@
 public class RecordToForward<K, V> {
   @Nullable private String childName;
   private Record<K, V> record;
+
+  public static <K, V> RecordToForward<K, V> from(
```

Comment:
Since the time will most like be System.currentTimeMillis() in majority of the cases, can we have another one without the timestamp argument?

Reply:
Added 2 more methods (one without timestamp, one with record directly)